application/safety_main.py

```diff
@@ -460,6 +465,8 @@ def retranslateUi(self, TransportationSafety):
         self.actionOpen_Config.setText(_translate("TransportationSafety", "Open Config ", None))
         self.actionNew_Project.setText(_translate("TransportationSafety", "New Project", None))
         self.actionNew_Project.setShortcut(_translate("TransportationSafety", "Ctrl+N", None))
+        self.actionFeedback.setText(_translate("TransportationSafety", "Send Feeback", None))
+        self.actionFeedback.setShortcut(_translate("TransportationSafety", "Ctrl+H", None))
```

Comment:
I don't know if CTRL+H is a great shortcut for this. I think it would be cleaner if it didn't automatically open a new browser window on an accidental command. Maybe just comment this out for now?